editor/project_settings_editor.cpp

```diff
@@ -722,7 +722,7 @@ ProjectSettingsEditor::ProjectSettingsEditor(EditorData *p_data) {
 
 	property_box = memnew(LineEdit);
 	property_box->set_placeholder(TTRC("Select a Setting or Type its Name"));
-	property_box->set_accessibility_name(TTRC("Setting Name"));
+	property_box->set_accessibility_name(TTRC("Select a Setting or Type its Name"));
```

Comment:
Would it make sense to fall back to the placeholder text?

Reply:
It already is using placeholder as fallback.

Reply:
No, just missed it, it’s already removed.